oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxNetworkIF.java

```diff
@@ -162,6 +194,7 @@ public boolean updateAttributes() {
         String collisionsPath = String.format("/sys/class/net/%s/statistics/collisions", getName());
         String rxDropsPath = String.format("/sys/class/net/%s/statistics/rx_dropped", getName());
         String ifSpeed = String.format("/sys/class/net/%s/speed", getName());
+        String ifAlias = String.format("/sys/class/net/%s/ifalias", getName());
```

Comment:
CI complains this local variable name shadows the class field name.  Must be a popular name. :)